mutagen/wave.py

```diff
@@ -151,10 +160,38 @@ def delete(self, filething):
 def delete(filething):
     """Completely removes the ID3 chunk from the RIFF/WAVE file"""
 
-    try:
-        _WaveFile(filething.fileobj).delete_chunk(u'id3')
-    except KeyError:
-        pass
+    for chunk in [u'id3', u'LIST']:
+        try:
+            _WaveFile(filething.fileobj).delete_chunk(chunk)
+        except KeyError:
+            pass
+
+
+def _read_tags_from_riff_list(fileobj):
+    """Read metadata tags from RIFF LIST tag and convert to ID3 format."""
+    wave_file = _WaveFile(fileobj)
+
+    if u"LIST" not in wave_file:
+        return None
+
+    list_chunk = wave_file[u"LIST"]
+    if list_chunk.name != u"INFO":
+        return None
+
+    wave_id3 = _WaveID3()
+    for chunk in list_chunk.subchunks():
+        try:
+            # Value is null-terminated and supposed to be ASCII, but not everyone
+            # respects that and puts unicode in there anyway
+            value = chunk.read()[:-1].decode("utf-8")
```

Comment:
utf-8 is not a safe choice either, though. foobar2k supports utf-8 in RIFF INFO. But Windows not, and will use Windows-1252 instead (or probably whatever encoding the localized version of Windows is using by default).

In the end it's a mess of course, because there is no way to indicate encoding. But maybe trying UTF-8 first and on `UnicodeDecodeError` retry with Windows-1252.